src/js/view/view.js

```diff
@@ -448,6 +448,8 @@ function View(_api, _model) {
         clickHandler.on({
             click: () => {
                 _this.trigger(DISPLAY_CLICK);
+                // Ensures that Firefox focuses the container not the video tag for aria compatibility
+                _getCurrentElement().focus();
```

Comment:
Does this still work when settingsMenuVisible() returns true?

Reply:
Great question! I opened the settings menu a few times, clicked the background and it does in fact keep the player container focused, should be all set there